dp/skaffold.yaml

```diff
@@ -106,7 +106,9 @@ profiles:
               extraEnv:
                 APP_CONFIG: "TestConfig"
             db_service:
-              enabled: false
+              enabled: true
```

Comment:
I'm afraid that this change requires also changing the setup in tests themselves, since setup in tests creates all models in set up and drops them in tear down, so it races with db services.
In fact this is exactly the reason, why it was disabled in the first place.
So to sum up, this change is fine, but only if accompanied by test setup change.